docs/usage.rst

```diff
@@ -140,6 +140,28 @@ Notice the transaction ``id``:
 
     In [0]: txid
 
+Check if the Transaction was sent successfully
+-----------------------------------------------
+
+After a couple of seconds, we can check if the transaction was validated in a
```

Comment:
I would write "included" rather than "validated" here. I realize you were just copying text from before, but we may as well fix it now.

Reply:
yes, that makes more sense!